simplq/src/styles/adminShareBar.module.scss

```diff
@@ -0,0 +1,8 @@
+@import 'utils';
+
+.main-button-content {
+  @include center-horizontally();
+  & > div {
+    margin: 1rem;
+  }
+}
```

Comment:
Can you put this in the adminPage css file. We don't want to have an entire file for just one class. One file per page is better.